### ISSUE TITLE
Fields initialized twice

### ISSUE THREAD

**Issue by kkrull** [kkrull/javaspec#2]
Fields in a spec class are initialized twice.  If that initialization has side effects, then the side effects happen twice.

Take the following example, which creates **2** temporary files but only deletes 1 of them.

```
@RunWith(JavaSpecRunner.class)
public class TestWithFile {
    private Closeable subject;
    class after_opening {
      private File path = File.createTempFile("DeletableOnlyWhenClosed", ".txt"); //RUNS TWICE

      Establish that = () -> subject = ...
      Because of = () -> subject.close();
      It closes_the_underlying_stream = () -> FileUtils.forceDelete(path);
    }
```

**Comment by kkrull** [comment]
Scheduled for the next release.

**Comment by kkrull** [comment]
Fixed in version 0.4.2.
